src/aics-image-viewer/shared/utils/controlPointsToLut.ts

```diff
@@ -126,7 +126,7 @@ export function initializeLut(
 }
 
 export function controlPointsToRamp(controlPoints: ControlPoint[]): [number, number] {
-  if (controlPoints.length === 1 || controlPoints.length === 3) {
+  if (controlPoints.length < 1 || controlPoints.length === 3) {
```

Comment:
handle crash if empty control points are passed to this method

Reply:
did you want <= 1 here? what happens for length=1?

Reply:
Yup, good catch